chef_master/source/deprecations_locale_lc_all.rst

```diff
@@ -0,0 +1,42 @@
+=======================================================================
+Deprecation: Deprecation of lc_all from locale resource (CHEF-27)
+=======================================================================
+`[edit on GitHub] <https://github.com/chef/chef-web-docs/blob/master/chef_master/source/deprecations_locale_lc_all.rst>`__
+
+.. tag deprecations_locale_lc_all
+
+Setting the ``LC_ALL`` variable is NOT recommended. As a system-wide setting, ``LANG`` should provide the desired behavior. ``LC_ALL`` is intended to be used for temporarily troubleshooting issues rather than an everyday system setting.
+Changing ``LC_ALL`` can break Chef's parsing of command output in unexpected ways. Use one of the more specific ``LC_`` properties as needed.
+This deprecation warning was added in Chef 14.12. Support for property ``lc_all`` will be removed for Chef 16.0.
+
+.. end_tag
+
+
+Remediation
+=======================================================================
+
+Setting *LC_* variables varies by platform, but common locations to configure *LC_* variables are:
```

Comment:
@btm I'm not finding other variables with the prefix ``LC_`` anywhere else in the docs. There's only ``lc_all`` or ``LC_ALL`` referenced ([see](https://docs.chef.io/resource_locale.html)). Is there something we need to update?

Reply:
Yes, It'll be updated later ! I'm working on it.

Reply:
Hey @mjingle , Please take a look at [this](https://github.com/chef/chef-web-docs/pull/1862). Here, I've mentioned the changes related to this statement. [Code changes](https://github.com/chef/chef/pull/8324) are also completed and under review.
Do let me know in case of any issues/concerns.